fableous-fe/src/App.tsx

```diff
@@ -139,7 +139,10 @@ export default function App() {
                 <InjectAxiosRespInterceptor />
                 <ThemeProvider theme={theme}>
                   <Navbar />
-                  <Container className="flex flex-col flex-1 pt-5 pb-12">
+                  <Container
+                    className="flex flex-col flex-1 pt-5 pb-12"
+                    disableGutters
```

Comment:
@wahyuadt @daystram was running localhost of master branch and I notice there is a minor UI problem, see screenshots below, after testing it is caused by disableGutters. Is it needed for anything specific? if yes, can find alternative, if not we can consider removing disableGutters

join room with disableGutters, no left padding

![image](https://user-images.githubusercontent.com/29034900/134791776-6e7d520f-3c43-40b5-ab10-ff27540e6d29.png)
join room without disableGutters

![image](https://user-images.githubusercontent.com/29034900/134791810-1de7736e-ebd1-47c3-bf7b-8686a088f01e.png)

drawing screen with disableGutter, white background on right and bottom edge of screen, makes page scrollable in x and y axis

![image](https://user-images.githubusercontent.com/29034900/134791766-1a04b959-9810-434e-a051-90c7427f862f.png)

drawing screen without disableGutter

![image](https://user-images.githubusercontent.com/29034900/134791803-b2cad5fa-9fdd-4d42-a02b-facd6653c220.png)